RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
Thanks @ianna. Just to make sure that I understand correctly - I tried the following.
1. I changed the name of our (python) `XMLIdealGeometryESSource` instance to `XMLIdealGeometryESSource_CTPPS`
2. This makes `XMLIdealGeometryESSource` save `DDCompactView` to ES with label `XMLIdealGeometryESSource_CTPPS`
3. When looking for our `DDCompactView` in ES, I do:
```
    edm::ESHandle<DDCompactView> idealCV;
    iRecord.getRecord<IdealGeometryRecord>().get("XMLIdealGeometryESSource_CTPPS", idealCV);
```
This seems to work. If you like it, let me know and I update the PR.